### ISSUE TITLE
UI - encryption/signature badges to support multiline

### ISSUE THREAD

**Issue by tomholub** [FlowCrypt/flowcrypt-ios#1086]
> Got this for FlowCrypt/flowcrypt-ios/Core/source/assets/compat/mime-email-plain-signed-detached.txt. Signature error looks too long.

The error is not necessarily too long, but he visual should be fixed so that the badge flows on multiple lines. I'll file a separate issue for that.

_Originally posted by @tomholub in https://github.com/FlowCrypt/flowcrypt-ios/issues/1020#issuecomment-978133605_

**Comment by tomholub** [comment]
![image](https://user-images.githubusercontent.com/6306961/143296245-08c408a1-b5c0-45d4-97da-8cc373f0f7e2.png)

**Comment by DenBond7** [comment]
@tomholub Maybe it will be better just to show `can't verify signature`. After that, a user can click on that badge and we will show a dialog with the details. If the text will be really big the UI will look a little ugly.

One more thing - I think we can do all these badges clickable. And show details after clicking on them.

For example:

encrypted -> this message was encrypted for AB...OF and BB...RH fingerprints
not encrypted -> it's a regular message. Not protected.
signed -> signed by AB...OF

and so on...

Like Thunderbird does it

![image](https://user-images.githubusercontent.com/2863246/145086393-cbae671b-aace-436c-97d6-6476a46a2943.png)

**Comment by tomholub** [comment]
Yes, I suppose we could do that.

**Comment by tomholub** [comment]
When tapped, it could produce a modal with more details, which is simple enough to do in code.

**Comment by tomholub** [comment]
@ioanmo226 I've just assigned you a few :-) I want to still think more about this one

**Comment by ioanmo226** [comment]
ok

**Comment by ioanmo226** [comment]
@tomholub Have you decided yet?
@sosnovsky Could you answer in case Tom is busy?

**Comment by sosnovsky** [comment]
I think we can use our existing block with message details for showing signature verification errors:

<img width="284" alt="Screenshot 2023-09-18 at 22 15 49" src="https://github.com/FlowCrypt/flowcrypt-ios/assets/1062093/7fe208d2-6038-45e6-92dc-a4ae55bddc77">

So badge will show just `cannot verify signature` text, while `no public key ... for email ...` text will go to the bottom of message details block. And this block should become visible after tapping on signature badge (currently it's shown only after tap on message recipients).

@ioanmo226 what do you think about such solution?

**Comment by ioanmo226** [comment]
Sounds good!

**Comment by ioanmo226** [comment]
Simplest solution for above issue
